src/freenet/client/filter/FilterUtils.java

```diff
@@ -397,9 +397,9 @@ else if(value.length()==7)
 			{
 
 				try{
-					Integer.valueOf(value.substring(1,3),16).intValue();
-					Integer.valueOf(value.substring(3,5),16).intValue();
-					Integer.valueOf(value.substring(5,7),16).intValue();
+                    Integer.valueOf(value.substring(1, 3), 16);
+                    Integer.valueOf(value.substring(3, 5), 16);
+                    Integer.valueOf(value.substring(5, 7), 16);
```

Comment:
indentation, too.

Reply:
Should be fixed in [b32d928]

Thank you